spec/models/manageiq/providers/azure/cloud_manager/provision/cloning_spec.rb

```diff
@@ -1,10 +1,18 @@
 require 'azure-armrest'
 
 describe ManageIQ::Providers::Azure::CloudManager::Provision::Cloning do
-  let(:provision) { ManageIQ::Providers::Azure::CloudManager::Provision.new }
+  let(:template) { FactoryBot.create(:azure_image, ext_management_system: ems) }
+  let(:provision) { FactoryBot.create(:miq_provision_azure, :options => {:src_vm_id => template.id}, :miq_request => miq_request) }
+  let(:configuration) {double}
```

Comment:
```suggestion
  let(:configuration) {double('azure armrest configuration'}
```

Or something to indicate what the double is for